src/ngMessageFormat/messageFormatCommon.js

```diff
@@ -34,7 +34,7 @@ function parseTextLiteral(text) {
   parsedFn['$$watchDelegate'] = function watchDelegate(scope, listener, objectEquality) {
     var unwatch = scope['$watch'](noop,
         function textLiteralWatcher() {
-          if (isFunction(listener)) { listener.call(null, text, text, scope); }
+          if (isFunction(listener)) { listener(text, text, scope); }
```

Comment:
Not sure about this one.
I believe that the point is that we actively want to ensure that we do not leak any `this` value through to the user provided listener function.
I think we might even have a test in `ngParse` for this...